### PR TITLE
Notification improvement date fns

### PR DESCRIPTION
**Pull request by karan-push**

### CODE REVIEW

File: src/components/notification/index.tsx [hunk 1]
```diff
@@ -230,10 +230,7 @@ const ViewNotificationItem: React.FC<NotificationItemProps> = ({
           <Pool>
             <PoolShare theme = {theme}>
               {timeStamp
-                ? moment
-                    .utc(parseInt(timeStamp) * 1000)
-                    .local()
-                    .format("DD MMM YYYY | hh:mm A")
+                ? format(new Date(Number(timeStamp) * 1000), 'dd MMM yyyy | hh:mm a')
```
Comment (subhranshudas) [inline src/components/notification/index.tsx]: lets move out this calculation to a utils function called `getFormattedDate`

Comment (subhranshudas) [inline src/components/notification/index.tsx]: can we paste some screenshot of the old date string vs what we get with this transformation?

**Comment by karan-push** [comment]
Notice the timestamp at the bottom right corner

with moment:
image

after replacing moment with date-fns:
image

**Comment by subhranshudas** [comment]
all good, you can merge Karan

**Pull request merged**